### ISSUE TITLE
[Task] question-answering model support

### ISSUE THREAD

**Issue by zhenchaoni** [microsoft/winml-cli#270]
## Overview

Extractive question-answering models take a context passage and a natural language question, then identify the answer span directly within the passage. These are reader/comprehension models (BERT, RoBERTa, DeBERTa, DistilBERT, ELECTRA variants) fine-tuned on SQuAD-style datasets.

The goal of this task is to verify that QA models exported through ModelKit (`wmk build`) produce accuracy that matches the HuggingFace PyTorch baseline — i.e., no accuracy regression from ONNX conversion or EP-specific quantization.

## Agent Scenarios

- **Document Q&A agent**: answer user questions grounded in a retrieved passage (contracts, manuals, reports) without hallucination — model reads the doc and extracts the answer span
- **Customer support agent**: resolve support queries by reading from a knowledge base article and returning the exact answer sentence
- **Research assistant agent**: extract specific facts (dates, names, figures) from long-form papers or reports on demand
- **On-device copilot agent**: run low-latency extractive QA locally on NPU for privacy-sensitive documents that cannot be sent to a cloud API

## ModelKit Integration

```
wmk config → wmk build (ONNX export) → wmk perf → wmk eval
```

Accuracy metric: Exact Match (EM) and F1 against SQuAD2.0, compared to HuggingFace PyTorch baseline. Regression threshold TBD.

## Acceptance Criteria

- [ ] ahotrod/electra_large_discriminator_squad2_512
- [ ] deepset/roberta-base-squad2
- [ ] google-bert/bert-large-uncased-whole-word-masking-finetuned-squad
- [ ] timpal0l/mdeberta-v3-base-squad2
- [ ] distilbert/distilbert-base-cased-distilled-squad
- [ ] deepset/bert-large-uncased-whole-word-masking-squad2
- [ ] monologg/koelectra-small-v2-distilled-korquad-384
- [ ] tau/splinter-base
- [ ] deepset/tinyroberta-squad2
- [ ] distilbert/distilbert-base-uncased-distilled-squad
